MonMaraicher/Features/FarmerDetail/FarmerDetailsView.swift

```diff
@@ -114,6 +115,22 @@ private extension FarmerDetailsView {
         .font(.headline)
     }
 
+    private var phoneButton: some View {
+        VStack {
+            if viewModel.phoneNumber == nil {
+                Image(systemName: "phone.fill")
+            } else {
+                if let number = viewModel.phoneNumber {
+                    if let url = URL(string: "tel:\(number)") {
```

Comment:
You can combine 2 if let in one like so:

`if let phoneNumber = viewModel.phoneNumber, let url = URL(...) {...}`

Reply:
You could also make a computed property in you viewModel to get directly the phoneCallURL and move your if condition inside your viewModel